OrbitVulkanLayerClientGgp/OrbitCaptureClientLayer.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <assert.h>
+#include <string.h>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitVulkanLayerClientGgp/DispatchTable.h"
+#include "absl/base/casts.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/synchronization/mutex.h"
+#include "vulkan/vk_layer.h"
```

Comment:
Can you add the following packages to the bootstrap-orbit.sh (~line 29) and to the docker files (e.g. `third_party/conan/docker/Dockerfile.clang7 `) 
```
libvulkan-dev vulkan-validationlayers-dev
```

Reply:
Installing dev packages for dependencies locally on the build machines is somewhat of a last resort. We do this for qt to build the Linux UI. It is ok-ish to get this running experimentally but we should get this into conan/cmake since Vulkan is something we will make use of.
As I wrote in a chat to Cristina - I am mildly puzzeled why this even compiles on the CI. It seems as if the dev package are already installed accidentally.

Reply:
>Installing dev packages for dependencies locally on the build machines is somewhat of a last resort. We do this for qt to build the Linux UI. It is ok-ish to get this running experimentally but we should get this into conan/cmake since Vulkan is something we will make use of.
As I wrote in a chat to Cristina - I am mildly puzzeled why this even compiles on the CI. It seems as if the dev package are already installed accidentally.

I discussed this with Henning a while ago, and we decided this will be the easiest and yet robust solution. This is also why the CI has these packages btw.

Reply:
But adding @beckerhe here.

Reply:
>I discussed this with Henning a while ago, and we decided this will be the easiest and yet robust solution. This is also why the CI has these packages btw.

Let's go with the dev-packages then for now. I'll chat with Henning to see if there is something to improve here.